config/docker/clang-base/Dockerfile

```diff
@@ -9,7 +9,11 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     binutils-aarch64-linux-gnu \
     binutils-arm-linux-gnueabihf \
     binutils-riscv64-linux-gnu \
-    binutils
+    binutils \
+    dwarves \
+    lz4 \
+    python3 \
+    python
```

Comment:
Why is this needed here?

Reply:
It's not clear to me why this is needed in the Clang images and not the GCC ones.